generator/mavschema.xsd

```diff
@@ -135,10 +134,29 @@
   </xs:restriction>
 </xs:simpleType>
 
+<!-- definition param elements attributes (like the ones used on MAV_CMD for example) -->
+<xs:attribute name="index" type="xs:unsignedByte"/>                    <!-- parameter index -->
+<xs:attribute name="label" type="xs:string"/>                          <!-- parameter label (name) -->
+<xs:attribute name="decimalPlaces" type="xs:unsignedByte"/>            <!-- parameter number of decimal places to be displayed in the GUI -->
+<xs:attribute name="increment" type="xs:float"/>                       <!-- parameter increment -->
+<xs:attribute name="minValue" type="xs:float"/>                        <!-- parameter minimum value -->
+<xs:attribute name="maxValue" type="xs:float"/>                        <!-- parameter maximum value -->
+
+<!-- definition entry elements attributes (like the ones used on MAV_CMD for example) -->
+<xs:attribute name="hasLocation" type="xs:boolean" default="true"/>    <!-- entry has lat/lon/alt location -->
+<xs:attribute name="isDestination" type="xs:boolean" default="true"/>  <!-- entry is a destination -->
+
 <!-- definition of complex elements -->
 <xs:element name="param">
     <xs:complexType mixed="true">
         <xs:attribute ref="index" use="required"/>
+        <xs:attribute ref="label"/>
+        <xs:attribute ref="units"/>
+        <xs:attribute ref="enum" />
+        <xs:attribute ref="decimalPlaces"/>
```

Comment:
I guess this should be replaced with `print_format`. That way it reuses an existing concept. There is no need to introduce a new one IMHO.